modules/imgcodecs/test/test_gif.cpp

```diff
@@ -366,6 +366,56 @@ TEST(Imgcodecs_Gif, encode_IMREAD_GRAYSCALE) {
     EXPECT_EQ(decoded.channels(), 1);
 }
 
+typedef testing::TestWithParam<int> Imgcodecs_Gif_GIF_LOOP;
+TEST_P(Imgcodecs_Gif_GIF_LOOP, imencode)
+{
+    int loopCount = GetParam();
+
+    vector<int> param;
+    param.push_back(cv::IMWRITE_GIF_LOOP);
+    param.push_back(loopCount);
+
+    vector<cv::Mat> src;
+    for(int n = 1; n <= 5 ; n ++ )
+    {
+        cv::Mat frame(64, 64, CV_8UC3, cv::Scalar::all(0));
+        cv::putText(frame, cv::format("%d", n), cv::Point(0,64), cv::FONT_HERSHEY_PLAIN, 4.0, cv::Scalar::all(255));
+        src.push_back(frame);
+    }
+
+    bool ret = false;
+    vector<uint8_t> buf;
+#if 0
+    // To output gif image for test.
+    EXPECT_NO_THROW(ret = imwrite(cv::format("gif_loop-%d.gif", loopCount), src, param));
+    EXPECT_TRUE(ret);
+#endif
+    EXPECT_NO_THROW(ret = imencode(".gif", src, buf, param));
+    EXPECT_TRUE(ret);
+
+    std::vector<uint8_t> netscape = {'N','E','T','S','C','A','P','E'};
+    auto pos = std::search(buf.begin(), buf.end(), netscape.begin(), netscape.end());
+    if(loopCount < 0) {
+        EXPECT_EQ(pos, buf.end()) << "Netscape Applicatioin Block should not be included if IMWRITE_GIF_LOOP < 0";
```

Comment:
Netscape Application

Reply:
I fixed it, thank you for your comment !